src/test/models/datafilterargs.spec.ts

```diff
@@ -18,6 +18,8 @@ describe('DataFilterArgsTest', () => {
     expect(args.buildFilterString()).toEqual("?start=2016-09-13T20:29:13.825Z&end=2016-10-13T20:29:13.825Z");
     args.page = 2;
     expect(args.buildFilterString()).toEqual("?start=2016-09-13T20:29:13.825Z&end=2016-10-13T20:29:13.825Z&page=2");
+    args.streamerId = 3793;
+    expect(args.buildFilterString()).toEqual("?start=2016-09-13T20:29:13.825Z&end=2016-10-13T20:29:13.825Z&page=2&streamer_id=3793");
```

Comment:
You also need to at least manually test the URL to ensure the cloud will return what you expect. You would have seen that `streamer_id` is not correct for specifying the beginning of a range